### PR TITLE
fix: update db schema update constraint

### LINKED ISSUE #44: Patch db schema to include update cascade

### PR DESCRIPTION
**Pull request by pyphilia**
allows rows to update if item moves

close #44

### CODE REVIEW

File: db-schema.sql [hunk 1]
```diff
@@ -1,6 +1,6 @@
 CREATE TABLE "recycled_item" (
   "id" uuid PRIMARY KEY UNIQUE NOT NULL DEFAULT uuid_generate_v4(), -- generated programatically and passed on insertion
-  "item_id" uuid UNIQUE NOT NULL REFERENCES "item" ("id") ON DELETE CASCADE,
+  "item_id" uuid UNIQUE NOT NULL REFERENCES "item" ("id") ON DELETE CASCADE ON UPDATE CASCADE,
```
Comment (mochiruntime) [inline db-schema.sql]: Info: what are the semantics of this change? I'm not I understand the purpose: it should only affect a row in recycled item if the id of the foreign item is affected, e.g. if it changed from UUID x to UUID y. However, this should never happen right? IDs are never re-assigned, even on move right?

**Review by mochiruntime — CHANGES_REQUESTED** [review]
@pyphilia I'm not clear why this change would actually solve the issue

File: db-schema.sql (hunk 1 above)
Comment (pyphilia) [reply db-schema.sql]: You're completely right, I've mixed up the line... actually I think we don't use item id anymore but well...

**Review by mochiruntime — APPROVED** [review]
LGTM :cactus:

**Pull request merged**